cms/templates/js/previous-video-upload-list.underscore

```diff
@@ -5,7 +5,7 @@
         <%- gettext("Download available encodings (.csv)") %>
       </a>
     </div>
-    <table class="assets-table">
+    <table class="assets-table" aria-live="polite">
```

Comment:
So, I mocked up some prototypes and found that aria-live on table elements is not supported well at all.  The only way I could get the live regions to work is to use a ```role="alert"```. The downside to that approach is that it reads out the entire table (because with role of alert, aria-atomic is true)

What I found even more confusing was that if I added ```aria-live="assertive aria-atomic="true"```  (Which is exactly the same as adding ```role="alert"``` ) The live region did not work at all.  Let's omit it here on the table all together.